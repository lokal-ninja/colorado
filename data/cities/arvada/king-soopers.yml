id: 2199853964
type: node
latitude: 39.84342
longitude: -105.08349
postcode: ""
city: Arvada
street: ""
housenumber: 
phone: "+1 303-420-4611"
opening_hours: "Mo-Su 06:00-23:00"
website: ""