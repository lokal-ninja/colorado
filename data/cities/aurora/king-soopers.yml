id: 1272751961
type: node
latitude: 39.6550819
longitude: -104.7761461
postcode: "80013"
city: Aurora
street: "East Hampden Avenue"
housenumber: 18211
phone: ""
opening_hours: ""
website: "https://www.kingsoopers.com/"
