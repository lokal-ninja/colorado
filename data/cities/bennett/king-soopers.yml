id: 266752370
type: way
latitude: 39.7434608
longitude: -104.4360733
postcode: "80102"
city: Bennett
street: "South 1st Street"
housenumber: 1045
phone: "(303) 644-5070"
opening_hours: "Su-Sa 05:00-23:59"
website: "https://www.kingsoopers.com/stores/details/620/00112"
