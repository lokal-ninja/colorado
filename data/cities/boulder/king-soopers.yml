id: 25757229
type: node
latitude: 39.9834
longitude: -105.25147
postcode: "80305"
city: Boulder
street: "Table Mesa Drive"
housenumber: 3600
phone: "+1-303-499-4004"
opening_hours: "Mo-Su 06:00-23:00"
website: "https://www.kingsoopers.com/stores/details/620/00033"