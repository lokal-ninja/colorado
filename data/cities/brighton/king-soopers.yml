id: 220973177
type: node
latitude: 39.97008
longitude: -104.81662
postcode: "80601"
city: Brighton
street: East Bromley Lane
housenumber: "500"
phone: ""
opening_hours: ""
website: ""
