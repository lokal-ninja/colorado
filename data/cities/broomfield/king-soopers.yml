id: 5447672248
type: node
shop: convenience
latitude: 39.932390600000005
longitude: -105.0902379
postcode: "80020"
city: Broomfield
street: "US 287"
housenumber: 1120
phone: ""
opening_hours: ""
website: ""
