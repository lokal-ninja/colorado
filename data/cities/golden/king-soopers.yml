id: 92725310
type: node
latitude: 39.73974
longitude: -105.19347
postcode: "80401"
city: Golden
street: South Golden Road
housenumber: "17171"
phone: ""
opening_hours: ""
website: ""
